spec/support/helpers/registration.rb

```diff
@@ -0,0 +1,18 @@
+module Helpers
+  module Registration
+    def register_user(username, email)
+      visit new_user_registration_path
+      
+      fill_in 'user[username]', with: username
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.